ios/Classes/TiNfcVTagTechnology.m

```diff
@@ -9,4 +9,701 @@
+
+  [[self.tagProxy asNFCISO15693Tag] resetToReadyWithRequestFlags:requestFlags
+                                               completionHandler:^(NSError *_Nullable error) {
+                                                 if (![self _hasListeners:@"didResetToReadyWithRequestFlags"]) {
+                                                   return;
+                                                 }
+                                                 [self fireEvent:@"didResetToReadyWithRequestFlags"
+                                                      withObject:@{
+                                                        @"errorCode" : error != nil ? NUMINTEGER([error code]) : [NSNull null],
+                                                        @"errorDescription" : error != nil ? [error localizedDescription] : [NSNull null],
+                                                        @"errorDomain" : error != nil ? [error domain] : [NSNull null]
+                                                      }];
+                                               }];
+}
+
+- (void)selectWithRequestFlags:(id)args
+{
+  NSNumber *requestFlagsValue = [[args firstObject] valueForKey:@"requestFlags"];
+  uint8_t requestFlags = [requestFlagsValue unsignedCharValue];
+
+  [[self.tagProxy asNFCISO15693Tag] selectWithRequestFlags:requestFlags
+                                         completionHandler:^(NSError *_Nullable error) {
+                                           if (![self _hasListeners:@"didSelectWithRequestFlags"]) {
+                                             return;
+                                           }
+                                           [self fireEvent:@"didSelectWithRequestFlags"
+                                                withObject:@{
+                                                  @"errorCode" : error != nil ? NUMINTEGER([error code]) : [NSNull null],
+                                                  @"errorDescription" : error != nil ? [error localizedDescription] : [NSNull null],
+                                                  @"errorDomain" : error != nil ? [error domain] : [NSNull null]
+                                                }];
+                                         }];
+}
+
+- (void)stayQuiet:(id)args
+{
+  [[self.tagProxy asNFCISO15693Tag] stayQuietWithCompletionHandler:^(NSError *_Nullable error) {
+    if (![self _hasListeners:@"didStayQuiet"]) {
+      return;
+    }
+    [self _hasListeners:@"didStayQuiet"];
+  }];
+}
+- (void)customCommandWithRequestFlag:(id)args
+{
+  NSNumber *requestFlagsValue = [[args firstObject] valueForKey:@"requestFlags"];
+  uint8_t requestFlags = [requestFlagsValue unsignedCharValue];
+
+  NSNumber *customCommandCode = [[args firstObject] valueForKey:@"customCommandCode"];
+
+  NSArray *customRequestParameters = [[args firstObject] valueForKey:@"customRequestParameters"];
+  unsigned int dataValue[customRequestParameters.count];
```

Comment:
Same here, since it is repetitive code block, better to create a separate methods with name like **arrayToData**.

Reply:
Done